Algeria.json

```diff
@@ -1958,7 +1973,7 @@
             {
                "arabic": "الجزائر",
                "french": "Alger",
-               "matricule": "16",

```

Comment:
You might have accidentally deleted the `matricule` attribute here. Please correct it :wink:

Reply:
Ouch! Sorry, I'll fix it right now